tools/webpack-plugin/src/models/AssetStats.ts

```diff
@@ -0,0 +1,7 @@
+import { UploadResult } from '@backtrace/sourcemap-tools';
+
+export interface AssetStats {
+    debugId?: string;
```

Comment:
From a prototyping perspective, it's ok to have everything nullable, but is that the case in the end? The asset stats must have debugId. I would even say more - if we print anything to the user, the model also should have a path to a file for unexpected errors with unknown error messages from the user's point of view without a path to a field.